create-convex/src/index.ts

```diff
@@ -134,19 +134,19 @@ async function init() {
         },
         {
           type: (framework) => {
-            if (framework.name === "other") {
+            if (argTemplate) {
+              return null;
+            }
+
+            if (framework?.name === "other") {
               throw new Error(
                 red("✖") +
                   " Follow one of the quickstarts at " +
                   bold("https://docs.convex.dev/quickstarts")
               );
             }
 
-            return argTemplate
-              ? null
-              : framework.name === "bare"
-              ? null
-              : "select";
+            return framework?.name === "bare" ? null : "select";
```

Comment:
There is an issue with `npm create convex bla -- -t bare`, although I'm not sure this is the right fix.

Reply:
Edit: This looks good